Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -1899,6 +1983,117 @@ def get_notable_sequence_event_types(client: Client, args: Dict[str, str]) -> Tu
+
+    incidents = []
+    raw_response = dict()
+
+    if incident_ids:
+        for incident_id in incident_ids:
+            raw_response = client.get_single_incident(incident_id, username)
+            incidents.append(format_single_incident(raw_response))
+
+    else:
+        if any([query, incident_type, priority, status]):
+            raw_response = client.get_list_incidents(query, incident_type, priority, status, limit, page_size,
+                                                     page_number)
+            for incident in raw_response['incidents']:
+                incidents.append(format_single_incident(incident))
+        else:
+            return_error('One of the following params is a must: query, incident_type, priority, status')
+
+    entry_context = {'Exabeam.Incident(val.incidentId && val.incidentId === obj.incidentId)': incidents}
+
+    human_readable = tableToMarkdown('Incidents list:', incidents)
+
+    return human_readable, entry_context, raw_response
+
+
+def fetch_incidents(client: Client, args: Dict[str, str]) -> Tuple[list, dict]:
+
+    last_run = demisto.getLastRun()
+    demisto.debug(f"Last run before the fetch run: {last_run}")
+    start_time, end_time = get_fetch_run_time_range(
+        last_run=last_run,
+        first_fetch=args.get('first_fetch', '3 days'),
+        look_back=1,
+        date_format='%Y-%m-%dT%H:%M:%S.%f',
+    )
+
+    demisto.debug(start_time, end_time)
+    start_time_as_milisecound, end_time_as_milisecound = order_time_as_milisecound_for_fetch(start_time, end_time)
+
+    incident_type = argToList(args.get('incident_type'))
+    priority = argToList(args.get('priority'))
+    status = argToList(args.get('status'))
+    limit = arg_to_number(args.get('max_fetch', 50))
+    q = {
+        "queryMap": {
+            "status": status,
+            "incidentType": incident_type,
+            "priority": priority,
+            "startedDate": [
+                start_time_as_milisecound,
+                end_time_as_milisecound
+            ],
+        },
+        "sortBy": "createdAt",
+        "sortOrder": "asc",
+        "idOnly": False,
+        "offset": 0,
+        "length": last_run.get('limit') or limit
+    }
+
+    resp = client.get_incidents(q)
+    incidents_res: List[dict] = resp.get('incidents', [])
+
+    incidents_filtered = filter_incidents_by_duplicates_and_limit(
+        incidents_res=incidents_res,
+        last_run=last_run,
+        fetch_limit=limit,
+        id_field='incidentId'
+    )
+
+    incidents: List[dict] = []
+    for incident in incidents_filtered:
+        incident.update({
+            'createdAt': datetime.fromtimestamp(
+                incident.get('baseFields', {}).get('createdAt') / 1000.0).strftime('%Y-%m-%dT%H:%M:%S.%f')})
+        incident = convert_all_unix_keys_to_date(incident)
+        incidents.append({
+            'Name': incident.get('name'),
+            'occurred': incident.get('baseFields', {}).get('createdAt'),
```

Comment:
is this a different value than the one we parse earlier?

Reply:
Yes, what I parse first is a timestamp by milliseconds, and here is a timestamp that our server is fine with it